core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -48,14 +48,19 @@ var ExplorationEditorPage = function() {
     .first();
```

Comment:
Please make the changes in this file in the equivalent webdriverIO file too

Reply:
done